src/ai/defaultai.cc

```diff
@@ -273,6 +274,26 @@ void DefaultAI::think() {
 
 	const int32_t delay_time = gametime.get() - taskPool.front().due_time.get();
 
+	// This portion of code keeps the speed of game so that FPS are kept within
+	// range 13 - 15, this is used for training of AI
+	if (game().is_auto_speed()) {
+		int32_t speed_diff = 0;
+		if (delay_time > 5000) {
+			speed_diff = -100;
+		} else if (delay_time < 1000) {
```

Comment:
Perhaps you can put there 1500.... to achieve bigger speeds

Reply:
ok will try. however reducing this to 4000 means we will get the 3rd job running only on rare occasions, as we start reducing speed before 4500 is reached,